main.js

```diff
@@ -34,65 +28,36 @@ function createOrder(text) {
     const editButton = newOrder.getEditButton()
     const editInput = newOrder.getEditInput()
 
-    orderText.addEventListener('click', event => {
+    order.setAttribute('is-done', newOrder.getIsDone())
+    order.setAttribute('todo-id', newId)
+    order.setAttribute('text', text)
 
-        if (!event.target.classList.contains('strike')) {
+    orderText.addEventListener('click', () => {
 
-            event.target.classList.remove('box__div')
-            event.target.classList.add('strike')
+        if (!orderText.classList.contains('strike')) {
+
+            orderText.classList.remove('box__div')
+            orderText.classList.add('strike')
```

Comment:
same here, you have all the info in the class, you don't need to update `is-done` attribute